Automattic/woocommerce-payments
# Disable platform checkout if subscription product is in cart

Fixes https://github.com/Automattic/woocommerce-payments/issues/3810

#### Changes proposed in this Pull Request
On the `/checkout` page, if there are any subscription products in the cart, then we do not load the platform checkout features. 

#### Testing instructions
1. Enable platform checkout feature.
2. Checkout product that are not subscription products. Confirm "email address", "Remember your details" shows up in checkout page.
![image](https://user-images.githubusercontent.com/572862/160017014-d3355f13-0f7c-4aba-9a57-2760c597d5be.png)
3. Create a subscription product, add it to cart and checkout
4. The "email address" at the top and "Remember your details" are no longer there.
![image](https://user-images.githubusercontent.com/572862/160017469-77caf14d-941f-49ab-a0e2-adb828e0b907.png)


**Post merge**
